README.md

```
@@ -105,6 +106,24 @@ sandbox@my-assistant:~$ openclaw agent --agent main --local -m "hello" --session
 
 ---
 
+## Troubleshooting
+
+### Docker Permission Denied
+If the installer or `onboard` command reports that Docker is not running, ensure your user has permission to access the Docker socket:
+```console
+$ sudo usermod -aG docker $USER
```

markdownlint-cli2 (0.21.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)